### ISSUE TITLE
Provide mediation fee margin to Raiden client

### ISSUE THREAD

**Issue by karlb** [raiden-network/raiden-services#569]
We don’t need margins for flat and prop fees. But we do want margins if imbalance fees are involved, even when different fees cancel each other out. So a good margin would be something like `sum(abs(imf) for imf in imbalance_fees) * 0.01`. The client can't know this value, so the PFS must provide it.

We don’t need to do it right now. We can add it to the PFS at any time, then we can make the client look for it and if it is present use it instead of the old margin.

**Comment by palango** [comment]
The alternative would be to send the individual fees for each hop to the client, so that it can adapt the calculation  itself.

**Comment by karlb** [comment]
Yes, that would work too if we send the fee components separately for each mediator like
```js
[
  {flat: 0, prop:123, imbalance:-50},
  {flat: 0, prop:0, imbalance:30},
]
```
